model_compression_toolkit/qat/keras/quantizer/quantization_dispatcher_builder.py

```diff
@@ -18,26 +18,34 @@
 from model_compression_toolkit.core import common
 from model_compression_toolkit.core.common.framework_info import FrameworkInfo
 from model_compression_toolkit import qunatizers_infrastructure as qi
-from model_compression_toolkit.qat.keras.quantizer.ste_rounding.symmetirc_ste import STEWeightQuantizer
+from model_compression_toolkit.qat.keras.quantizer.ste_rounding.symmetric_ste import STEWeightQuantizer, STEActivationQuantizer
 from model_compression_toolkit.qat.keras.quantizer.ste_rounding.uniform_ste import STEUniformWeightQuantizer
 
-METHOD2QUANTIZER = {qi.QuantizationMethod.SYMMETRIC: STEWeightQuantizer,
-                    qi.QuantizationMethod.POWER_OF_TWO: STEWeightQuantizer,
-                    qi.QuantizationMethod.UNIFORM: STEUniformWeightQuantizer}
+METHOD2WEIGHTQUANTIZER = {qi.QuantizationMethod.SYMMETRIC: STEWeightQuantizer,
+                          qi.QuantizationMethod.POWER_OF_TWO: STEWeightQuantizer,
+                          qi.QuantizationMethod.UNIFORM: STEUniformWeightQuantizer}
+
+
+METHOD2ACTQUANTIZER = {qi.QuantizationMethod.SYMMETRIC: STEActivationQuantizer,
+                       qi.QuantizationMethod.POWER_OF_TWO: STEActivationQuantizer}  # TODO: Add uniform quantization
 
 
 def quantization_dispatcher_builder(n: common.BaseNode,
                                     fw_info: FrameworkInfo,
-                                    method2quantizer: Dict[
-                                        qi.QuantizationMethod, qi.BaseKerasQuantizer] = METHOD2QUANTIZER) -> qi.KerasNodeQuantizationDispatcher:
+                                    method2weightquantizer: Dict[
+                                        qi.QuantizationMethod, qi.BaseKerasQuantizer] = METHOD2WEIGHTQUANTIZER,
+                                    method2actquantizer: Dict[
+                                        qi.QuantizationMethod, qi.BaseKerasQuantizer] = METHOD2ACTQUANTIZER
+                                    ) -> qi.KerasNodeQuantizationDispatcher:
```

Comment:
Maybe it's better to use None in the signature (and use default dicts if they are None)?